src/remote/remote.h

```diff
@@ -221,6 +221,27 @@ struct Rtr : public Firebird::GlobalStorage, public TypedHandle<rem_type_rtr>
 };
 
 
+struct RBlobInfo
+{
+	bool	valid;
+	UCHAR	blob_type;
```

Comment:
I believe all of us consider the `camelCase` field as more modern case convention of Firebird.
Many things there still use that underscores separators, but many things there are using camel case.
So why not use it in a new type?

Reply:
I usually prefer `camelCase` too (as well as `m_` prefix for class fields) but not in this case: these members named after the public constants `isc_info_blob_XXX` and such usage is very convenient, IMO.
Look at `RBlobInfo::getLocalInfo()` and `RBlobInfo::parseInfo()`, for example.